src/openrct2/ride/Ride.h

```diff
@@ -1097,7 +1097,7 @@ bool RideHasAdjacentStation(const Ride& ride);
 bool RideHasStationShelter(const Ride& ride);
 bool RideHasRatings(const Ride& ride);
 
-int32_t GetBoosterSpeed(ride_type_t rideType, int32_t rawSpeed);
+int32_t GetUnifiedBoosterSpeed(ride_type_t rideType, int32_t relativeSpeed);
```

Comment:
This name is a little better, but please add a doc-block to the function declarations (i.e. those in the .h files) in question.